server/src/api/v1/users/auth-user.ts

```diff
@@ -31,7 +30,7 @@ export async function loginUser(req: Request, res: Response) {
     if (user === null) {
         res.status(403).send(wrapResponse(false, { error: 'Unauthorized!' }));
     } else {
-        const token = jwt.sign({ id: user.id, email: user.email }, "unserKey", { expiresIn: calculatedExpiresIn });
+        const token = jwt.sign({ id: user.id, email: user.email, is_admin: user.is_admin}, 'JWT_HASH', { expiresIn: calculatedExpiresIn }); //TODO .env import
```

Comment:
```suggestion
        const token = jwt.sign({ id: user.id, email: user.email, is_admin: user.is_admin}, Vars.config.database.jwtSalt, { expiresIn: calculatedExpiresIn });
```

Import von Vars fehlt aber noch

Reply:
Und ja, das gehört nicht in db config aber ist so am einfachsten

Reply:
Ja, weiß ich. War nur ein zwischenstand, weil ich mich in teams angemeldet habe und keine Lösung mehr für den input da gefunden habe.
Aber danke :+1: